src/chainstate/stacks/db/unconfirmed.rs

```diff
@@ -237,6 +247,16 @@ impl UnconfirmedState {
         self.bytes_so_far += new_bytes;
         self.num_mblocks_added += num_new_mblocks;
 
+        // apply injected faults
```

Comment:
Can you separate this code out into a separate function to make it clearer that it is "testing" code?

Reply:
We can't :(  This is because the `#[cfg(test)]` pragma doesn't apply across crates.  The `blockstack_lib` crate is built without `#[cfg(test)]` when running integration tests.